oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -1086,6 +1116,10 @@ public Builder setLifetime(int lifetime) {
       return this;
     }
 
+    /**
+     * Sets the useJwtAccessWithScope flag. This flag will be ignored if universeDomain field is
+     * different from {@link Credentials.GOOGLE_DEFAULT_UNIVERSE}.
+     */
```

Comment:
I believe we have some logic regarding with using this flag in GAX: https://github.com/googleapis/sdk-platform-java/blob/c0e03cdce18e55f46e25f5aa71608416516cf4f9/gax-java/gax/src/main/java/com/google/api/gax/core/GoogleCredentialsProvider.java#L98-L102. qq, does this return a valid ServiceAccountCredentials that can be used or will it throw an exception somewhere later when the user tries to use the Credentials?

Reply:
It returns a valid ServiceAccountCredentials, its just this flag is ignored internally if universe is not default.

What the GAX code does is setting the value to true for GAPICs. Effectively enabling SelfSignedJwt. It will work as before in GDU. In other universes SelfSignedJWT is the only option no need to use the flag. But setting it will not cause any exception )

Reply:
Understood, thanks!